### ISSUE TITLE
TextView's font breaks when you type emoji

### ISSUE THREAD

**Issue by phlippieb** [CosmicMind/Material#838]
When using a TextView, if a user types emoji, the text after the emoji is in a different font. This works fine when using a normal UITextView.

To reproduce the issue, create a single application project with Material, and use this as the ViewController:

```
import UIKit
import Material

class ViewController: UIViewController {
    let uiTextView: UITextView = {
        let textView = UITextView()
        textView.borderWidth = 1
        return textView
    }()
    
    let mTextView: TextView = {
        let textView = TextView()
        textView.borderWidth = 1
        return textView
    }()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        view.addSubview(uiTextView)
        view.addSubview(mTextView)
    }
    
    override func viewDidLayoutSubviews() {
        super.viewDidLayoutSubviews()
        uiTextView.frame = CGRect(x: 32, y: 32, width: view.bounds.width - 64, height: 100)
        mTextView.frame = CGRect(x: 32, y: 164, width: view.bounds.width - 64, height: 100)
    }
}
```

As demonstrated in this screenshot, the UITextView (the first box) handles emoji correctly, while the Material TextView (the second box) does not.

![screen shot 2017-08-14 at 3 56 44 pm](https://user-images.githubusercontent.com/2232699/29274760-2ffa8eb0-8109-11e7-833d-7b462f64de43.png)

Slack seems to have the same issue: https://github.com/slackhq/SlackTextViewController/issues/536

I am using XCode 8.3.3, targeting iOS 9 and up (running iOS 10 in screenshot).

**Comment by phlippieb** [comment]
This seems to happen only when a TextView is initialized using the convenience no-parameter `init()`. Seemingly something about overriding how the layout manager is constructed breaks this. I can bypass the issue by constructing it as `TextView(textContainer: nil)`.

**Comment by danieldahan** [comment]
hmmm... I will have to take a look at this. Thank you, it is now in the queue.

**Comment by danieldahan** [comment]
This is now fixed in [Material 2.16.3](https://github.com/CosmicMind/Material/releases/tag/2.16.3). Thank you!
